tools/dev/loki-boltdb-storage-s3/compose-up.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$(cd `dirname $0` && pwd)
+
+SRC_DEST=${SCRIPT_DIR}/.src/
+# sync all sources for dlv
+rm -Rf ${SRC_DEST}
+mkdir ${SRC_DEST}
+for d in cmd pkg vendor clients
+do
+    cp -Rf ${SCRIPT_DIR}/../../../${d}/ ${SRC_DEST}/${d}/
```

Comment:
Are we running shellcheck on the Loki repo? I think variables in paths should be in quotes if I'm not mistaken.

Reply:
Not a bash fu so I trust you, I'll rebase to your PR.